src/components/rules/menu.tsx

```diff
@@ -73,3 +74,35 @@ export const RuleSetMenu = ({ id }: { id: string }) => {
     </Menu>
   );
 };
+
+export const FlowbiteRuleSetMenu = ({ id }: { id: string }) => {
```

Comment:
I know this probably just temp name, but I wouldn't use Flowbite in the menu name. Better to keep implementation details out of names. Maybe just call the old one OldRulesetMenu and this one RulesetMenu.